js/app/src/lite/custom-element/index.ts

```diff
@@ -0,0 +1,161 @@
+import { create, type Options } from "..";
```

Comment:
Note: This is a circular dependency ( `lite/index.js` <-> `lite/custom-element/index.ts`), but I left it as-is for now because fixing it will make a huge diff on `lite/index.js`).
Will do it after the first review or in another PR.

Reply:
Thanks, yep, the underlying reason `GradioLiteAppElement` is inside `bootstrap_custom_element` is to avoid the runtime error. It works well, but I thought someone would think it looks weird just in case.